website/content/docs/discovery/dns.mdx

```diff
@@ -96,47 +96,6 @@ pairs according to [RFC1464](https://www.ietf.org/rfc/rfc1464.txt).
 Alternatively, the TXT record will only include the node's metadata value when the
 node's metadata key starts with `rfc1035-`.
 
-## Alternative Domain
```

Comment:
It doesn't make sense to have this between the node lookup and service lookup sections.